executor/analyze.go

```diff
@@ -276,7 +276,7 @@ type AnalyzeColumnsExec struct {
 }
 
 func (e *AnalyzeColumnsExec) open() error {
-	ranges := ranger.FullIntNewRange()
+	ranges := ranger.FullIntNewRange(nil)
```

Comment:
@lamxTyler Do we need to choose unsigned int range and make it return in order?

Reply:
Yes, we should.

Reply:
Okay, i'll fix this in next commit.